recipes/libxsmm/build.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+make -j${CPU_COUNT} CXX=$CXX CC=$CC FC=$FC PREFIX=${PREFIX}
```

Comment:
Do you need to pass these variables through?

Reply:
PREFIX should also be passed through?

Reply:
Done.